python/samples/getting_started/00-getting-started.ipynb

```diff
@@ -16,6 +16,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "# Note: if using a Poetry virtual environment, do not run this cell\n",
     "!python -m pip install semantic-kernel==1.0.5"
```

Comment:
we should probably move to this: https://ipython.readthedocs.io/en/stable/interactive/magics.html#magic-pip so %pip install which installs in the current kernel, rather then using `!` which executes in shell...

Reply:
Great idea, let's do it. Will make the change now.